apps/bare-expo/scripts/start-android-e2e-test.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+CURR_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
+cd "${CURR_DIR}/../android"
+
+echo ''
+echo '🔌 Installing APKs'
+adb install -r app/build/outputs/apk/release/app-release.apk
+adb install -r app/build/outputs/apk/androidTest/release/app-release-androidTest.apk
+
+if [[ -n "${CI}" && -n "${GITHUB_ACTION}" ]]; then
+  echo ''
+  echo '💤 Waiting for GitHub CI emulator to warm up'
+  sleep 30
```

Comment:
Can we check if it's up and run `sleep` in a loop for a few seconds? Rather than keeping fingers crossed that it will warm up in 30 seconds 😅

Reply:
the code is later removed in #22216.
btw, `android-emulator-runner` action takes care of booted state already. the sleep 30 here is waiting for the emulator being idle a little bit. the phone/machine is typically pretty busy right after boot up.